### PR TITLE
community[minor]: Rename LLMonitor Callback Handler

### PR DESCRIPTION
**Pull request by hughcrt**
(Following #3614)

Hi,
We are renaming our product from LLMonitor to Lunary.ai.
This PR add the Lunary Callback Handler, but keeps the llmonitor one in order to avoid breaking changes.
Thanks!

### CODE REVIEW

File: libs/langchain-google-genai/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@langchain/google-genai",
-  "version": "0.0.3",
```
Comment (jacoblee93) [inline libs/langchain-google-genai/package.json:3]: Looks like this got pulled in by accident?

File: langchain/scripts/create-entrypoints.js [hunk 2]
```diff
@@ -360,6 +361,7 @@ const requiresOptionalDependency = [
   "tools/google_calendar",
   "tools/gmail",
   "callbacks/handlers/llmonitor",
+  "callbacks/handlers/lunary",
```
Comment (jacoblee93) [inline langchain/scripts/create-entrypoints.js:364]: We've deprecated new integration exports from `langchain` - can you remove and change examples to import from `@langchain/community/callbacks/handlers/lunary`?

File: langchain/callbacks/lunary.cjs [hunk 3]
```diff
@@ -0,0 +1 @@
+module.exports = require("../dist/callbacks/handlers/lunary.cjs");
```
Comment (jacoblee93) [inline langchain/callbacks/lunary.cjs:1]: These build artifacts should not be checked in, can you remove?

**Review by jacoblee93 — CHANGES_REQUESTED** [review]
Thanks! See comments, lots of structural changes recently:

**Comment by hughcrt** [comment]
Hi @jacoblee93,
Thanks for your prompt answer. I've updated the code by following your instructions.

**Review by hughcrt — COMMENTED** [review]
Done

**Comment by hughcrt** [comment]
Hi @jacoblee93 ,
Could you please check and confirm if the pull request can now be merged?

**Review by bracesproul — COMMENTED** [review]
looking good so far! I left a few comments, if you could resolve & re-request my review that would be great! Thank you!

**Comment by hughcrt** [comment]
Hi @jacoblee93 ,

I have made the requested changes. Thanks

File: docs/core_docs/docs/ecosystem/integrations/lunary.mdx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+<video controls width='100%' >
+  <source src='https://lunary.ai/videos/demo-annotated.mp4'/>
+</video>
+
+## Installation
+
+Start by installing the Lunary package in your project:
+
+```bash
```
Comment (bracesproul) [inline docs/core_docs/docs/ecosystem/integrations/lunary.mdx:19]:
```suggestion
```bash npm2yarn
```

File: langchain/callbacks/handlers/lunary.cjs
Comment (bracesproul) [inline langchain/callbacks/handlers/lunary.cjs:1]: Please delete these three files in `langchain/callbacks/handlers`

**Review by bracesproul — CHANGES_REQUESTED** [review]
Two small comments, also run `yarn format` from root then it's good to merge!

**Comment by hughcrt** [comment]
Hi @bracesproul,
Thanks for your review! I have deleted the build files, and ran `yarn format`

File: libs/langchain-community/src/callbacks/handlers/lunary.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export { LunaryHandler } from "lunary/langchain";
```
Comment (jacoblee93) [inline libs/langchain-community/src/callbacks/handlers/lunary.ts:1]: This is fine but just keep in mind we won't be able to update if refactors happen

**Comment by jacoblee93** [comment]
Hey @hughcrt, couldn't push changes to your branch but have opened #3895 and will merge that tonight.

Thanks for your patience, apologies for the delays!

**Comment by jacoblee93** [comment]
This is live in the latest @langchain/community

**Pull request closed without merging**